docs/development/methodology/trello.md

```diff
@@ -10,10 +10,10 @@ The name of the card should contain a user story that somebody will,
 eventually, have to do. This should be obtained from the requirements
 that the client has specified previously.
 
-The name of the card should be short but it needs to explain what is
-needed to be done. If there is some extra information that is needed
-to fully understand the task it should be written in the description
-of the card. In the same spirit, if there exists a document that is
```

Comment:
La palabra 'there' si va ahí, está gramaticalmente mal sin el there aunque suena medio mal. Mejor cambiarlo por "If a document exists that is related to the task"